stage.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 SUSE LLC
+
+import os
+import logging
+import shutil
+import sys
+import time
+
+from stagelib.envglobals import (printGlobal, CEPH_VERSION, BASEOS_NAME, BASEOS_TAG, BASEOS_REG,
```

Comment:
The parenthesis generates an invalid syntax in pyflakes

Reply:
From PEP8:
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.